internal/rego/rego.go

```diff
@@ -120,13 +121,18 @@ func loadRegoFiles(files []string) ([]File, error) {
 
 func getModuleRulesActions(module *ast.Module) []string {
 	var rulesActions []string
+	var previouslySeenActions []string
```

Comment:
I think we could just iterate through `rulesActions` before adding to it, rather than creating a new slice yeah?

Reply:
🤦